tokens/lib/tokens.json

```diff
@@ -3699,250 +3637,610 @@
+        },
+        "button": {
+          "default": {
+            "value": "{subdued-alt}",
             "type": "color"
           },
-          "error": {
-            "value": "{destructive}",
+          "hover": {
+            "value": "{subdued-alt}{alpha.hover-soft}",
             "type": "color"
           },
-          "info": {
-            "value": "{brand}",
+          "pressed": {
+            "value": "{subdued-alt}{alpha.hover-full}",
             "type": "color"
           },
-          "warning": {
-            "value": "{info}",
+          "active": {
+            "value": "{subdued-alt}{alpha.active}",
             "type": "color"
           }
-        },
-        "input-stepper": {
+        }
+      },
+      "st": {
+        "border": {
           "default": {
-            "value": "{brand}",
+            "value": "{st.color.border.default}",
             "type": "color"
           },
-          "disabled": {
-            "value": "{subdued-alt}",
+          "hover": {
+            "value": "{st.color.border.hover}",
             "type": "color"
           },
-          "readOnly": {
-            "value": "{subdued-alt}",
+          "error": {
+            "value": "{st.color.border.error}",
             "type": "color"
-          }
-        },
-        "select": {
-          "default": {
-            "value": "{brand}",
+          },
+          "wcag": {
+            "value": "{color.pink.500}",
+            "type": "color"
+          },
+          "warning": {
+            "value": "{st.color.border.warning}",
+            "type": "color"
+          },
+          "info": {
+            "value": "{st.color.border.info}",
+            "type": "color"
+          },
+          "success": {
+            "value": "{st.color.border.success}",
+            "type": "color"
+          },
+          "ai": {
+            "value": "{st.color.border.ai}",
             "type": "color"
           }
         },
-        "expandableMenu": {
-          "subduedDefault": {
+        "bg": {
+          "success": {
+            "inverted": {
+              "value": "{success}",
+              "type": "color"
+            },
+            "default": {
+              "value": "{st.color.bg.success.default}",
+              "type": "color"
+            },
+            "hover": {
+              "value": "{st.color.bg.success.default}{alpha.hover-full}",
+              "type": "color"
+            },
+            "pressed": {
+              "value": "{st.color.bg.success.default}{alpha.pressed}",
+              "type": "color"
+            },
+            "active": {
+              "value": "{st.color.bg.success.default}{alpha.active}",
+              "type": "color"
+            }
+          },
+          "error": {
+            "inverted": {
+              "value": "{destructive}",
+              "type": "color"
+            },
+            "default": {
+              "value": "{st.color.bg.error.default}",
+              "type": "color"
```

Comment:
The bg and border colors should not be called st.[token]... they should be bg.base, border.default etc. This is what they are called currently in app and it would otherwise be a breaking change